xml/deployment_preparation.xml

```diff
@@ -343,7 +362,9 @@ suse_caasp:
     </step>
     <step>
      <para>
-      Attach the <filename>cc-admin.img</filename> disk image as a secondary disk.
+      Attach the <filename>cc-admin.img</filename> image as a
+      secondary disk. For details about preparing the image, see <xref
```

Comment:
Secondary (hard) disk would probably also work but I was always attaching virtual DVD unit with mounted cc-admin|worker.iso file - in the ESX webui there is just a need to activate "Connect.." (on boot) checkbox for DVD unit with the mounted image.